baseapp/baseapp.go

```diff
@@ -133,6 +133,9 @@ type BaseApp struct { // nolint: maligned
 	// indexEvents defines the set of events in the form {eventType}.{attributeKey},
 	// which informs Tendermint what to index. If empty, all events will be indexed.
 	indexEvents map[string]struct{}
+
+	// fee handler
```

Comment:
NIT: Could probably remove this comment.  The variable name says the same thing.

Reply:
i'll take it